packages/augur-tools/src/flash/scripts.ts

```diff
@@ -2116,6 +2119,65 @@ export function addScripts(flash: FlashSession) {
     }
   });
 
+  flash.addScript({
+    name: 'sdk-server',
+    ignoreNetwork: true,
+    options: [
+      {
+        name: 'config-file',
+        abbr: 'f',
+        description: 'SDKConfiguration JSON file',
+      },
+      {
+        name: 'config',
+        abbr: 'c',
+        description: 'serialized SDKConfiguration JSON',
+      },
+    ],
+    async call(this: FlashSession, args: FlashArguments) {
+      const configFile = args.configFile as string;
+      const config = args.config as string;
+
+      let configuration: SDKConfiguration;
+      if (configFile && config) {
+        throw Error('Cannot specify both config-file and config');
+      } else if (configFile) {
+        configuration = JSON.parse(fs.readFileSync(configFile).toString())
+      } else if (config) {
+        configuration = JSON.parse(config);
+      } else {
+        // TODO support default network SDKConfiguration
+        throw Error('Must specify config-file or config')
+      }
+
+      const api = await startServer(configuration, this.account);
+      const app = createApp(api);
+
+      const endpointSettings: EndpointSettings = {
+        httpPort: 9003,
+        startHTTP: true,
+        httpsPort: 9004,
+        startHTTPS: true,
+        wsPort: 9001,
+        startWS: true,
+        wssPort: 9002,
+        startWSS: true,
+        ...(configuration.server || {})
+      };
+
+      const httpServer = endpointSettings.startHTTP && runHttpServer(app, endpointSettings);
+      const httpsServer = endpointSettings.startHTTPS && runHttpsServer(app, endpointSettings);
+      const wsServer = endpointSettings.startWS && runWsServer(api, app, endpointSettings);
+      const wssServer = endpointSettings.startWSS && runWssServer(api, app, endpointSettings);
+
+      await awaitUserInput('Running SDK server. Press ENTER to quit:\n');
```

Comment:
Instead of doing this -- we should probabaly just bind for SIGINT. Hitting any key to exit a running server is weird on any platform.

```suggestion
      process.on('SIGINT', () => {
          httpServer.close();
          httpsServer.close();
          wsServer.close();
          wssServer.close();
      });
```

Reply:
Oh, this applies for the other cases where we do this too. But don't these things gracefully shut down on SIGINT anyway? Is there some concern that they wont and we need to clean up? Or is this because of interactive mode?

Reply:
I did this because it felt wrong to `await` on the final never-ending promise, especially without telling the user what was going on. I agree that binding of SIGINT is a better solution.

Reply:
https://github.com/AugurProject/augur/issues/6386